Wrappers/Python/test/test_Operator.py

```diff
@@ -270,6 +270,55 @@ def test_FiniteDifference(self):
         w = G.direct(u)
         self.assertBlockDataContainerEqual(res, w)
         
+        # 2D       
+        M, N = 2, 3
+        ig = ImageGeometry(voxel_num_x=M, voxel_num_y=N, voxel_size_x=0.1, voxel_size_y=0.4)
+        x = ig.allocate('random')
+                
+        for i in range(2):
```

Comment:
if you define a list ['horizontal_y','horizontal_x'] and iterate over it with enumeration I think this will be slightly neater and easier to read.